webrtc.html

```diff
@@ -1862,6 +1880,52 @@
         </dl>
       </section>
     </section>
+    <section>
+      <h3>Session Negotiation Model</h3>
+      <p>Many changes to state of an <code><a>RTCPeerConnection</a></code> will
+        require communication with
+        the remote side via the signaling channel, in order to have the desired
+        effect. The app can be kept informed as to when it needs to do signaling,
+        by listening to the negotiationneeded event.</p>
+      <section>
+        <h4>Setting Negotiation-Needed</h4>
+        <p>If an operation is performed on an <code><a>RTCPeerConnection</a></code>
+          that requires signaling,
+          the connection will be marked as needing negotiation. Examples of such
+          operations include adding or stopping a track, or adding the first
+          data channel.</p>
+      </section>
+      <section>
+        <h4>Clearing Negotiation-Needed</h4>
+        <p>The negotiation-needed state is cleared when
+          <code>setLocalDescription</code> is
+          called (either for an offer or answer), and the supplied description
```

Comment:
I'm sort of wondering if  the match is needed. I'm thinking it can be cleared whenever setLocal is called. If the setLocal does not have some of the tracks, then those tracks would be removed ? Thoughts on this?